app/models/user.rb

```diff
@@ -468,6 +468,20 @@ def seed_aspects
     aq
   end
 
+  def send_welcome_message
+    unless AppConfig.settings.welcome_message? && AppConfig.admins.account? then return end
+    sender_username = AppConfig.admins.account.get
+    sender = User.find_by_username(sender_username)
```

Comment:
Aren't those finders deprecated in favour of `User.find_by(username: sender_username)`?